app/welcome/first-app/route.js

```diff
@@ -3,6 +3,11 @@ import { write } from '../../utils/storage';
 export var firstAppKey = '_aptible_firstAppData';
 
 export default Ember.Route.extend({
+  redirect: function(model){
+    if (model.appHandle && model.appHandle.length > 0) {
+      this.transitionTo('welcome.payment-info');
+    }
+  },
```

Comment:
This can be removed. appHandle will not be populated, we have removed the localStorage hijinks.